src/components/SearchOverlay.vue

```diff
@@ -84,7 +84,12 @@ async function search() {
     let applySalesPriceFacet = false;
     if (result.value?.facets?.items?.length === 3) {
         const salesPriceFacet = result.value?.facets.items[2] as PriceRangeFacetResult;
-        applySalesPriceFacet = salesPriceFacet && filters.value.price.length === 2 && Number(filters.value.price[0]) !== salesPriceFacet.available!.value?.lowerBoundInclusive || Number(filters.value.price[1]) !== salesPriceFacet.available!.value?.upperBoundInclusive;
+        applySalesPriceFacet = salesPriceFacet
+            && filters.value.price.length === 2
+            && (Number(filters.value.price[0]) !== salesPriceFacet.available!.value?.lowerBoundInclusive
+                && salesPriceFacet.available!.value?.lowerBoundInclusive !== 0)
+            || (Number(filters.value.price[1]) !== salesPriceFacet.available!.value?.upperBoundInclusive
+                && salesPriceFacet.available!.value?.upperBoundInclusive !== 0);
```

Comment:
Sweet, but can we do something to make the code a bit more readable?

Reply:
Check! I have split each check out into its own variable.